privaterelay/management/commands/update_fxrelay_allowlist_collection.py

```diff
@@ -0,0 +1,119 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import requests
+from kinto_http import Client, KintoException
+from kinto_http.patch_type import BasicPatch
+
+BUCKET = settings.KINTO_BUCKET
+COLLECTION = settings.KINTO_COLLECTION
+
+
+class Command(BaseCommand):
+    help = "Updates the Firefox Relay allowlist Kinto collection from a JSON source."
```

Comment:
I know that `kinto` is everywhere, since it's the engine that powers Remote Settings. But globally wherever you can I would recommend using *Remote Settings*.
```suggestion
    help = "Updates the Firefox Relay allowlist Remote Settings collection from a JSON source."
```